addons/advanced_ballistics/functions/fnc_handleFired.sqf

```diff
@@ -19,7 +19,7 @@
 //IGNORE_PRIVATE_WARNING ["_unit", "_weapon", "_muzzle", "_mode", "_ammo", "_magazine", "_projectile", "_vehicle", "_gunner", "_turret"];
 TRACE_10("firedEH:",_unit,_weapon,_muzzle,_mode,_ammo,_magazine,_projectile,_vehicle,_gunner,_turret);
 
-if (!(_ammo isKindOf "BulletBase")) exitWith {};
+if (!_ammo isKindOf "BulletBase") exitWith {};
```

Comment:
order of operations
!<string> happens first and won't be valid

Reply:
Fixed in newest commit